go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -487,8 +487,7 @@ func (erp *EmergencyReparenter) reparentReplicas(
 	tabletMap map[string]*topo.TabletInfo,
 	statusMap map[string]*replicationdatapb.StopReplicationStatus,
 	opts EmergencyReparentOptions,
-	waitForAllReplicas bool,
-	populateReparentJournal bool,
+	intermediateReparent bool,
```

Comment:
We should add a comment regarding what this parameter means as it's not obvious (to me anyway) from the name.